styles.css

```diff
@@ -493,3 +493,301 @@ textarea {
 .contact-form-end-image {
   margin-left: 100px;
 }
+
+.desktop-navigation {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    width: 100%;
+  }
+
+  .navigation {
+    display: none;
+  }
+
+  .desktop-navigation {
+    display: flex;
+    margin: 25px 154px 0 0;
+  }
+
+  .desktop-navigation ul a {
+    text-decoration: none;
+  }
+
+  .logo {
+    margin: 25px auto 0 154px;
+  }
+
+  main {
+    background-image: none;
+    background-image: url("images/desktop-Header-bg.svg"), url("images/desktop-header-background-shape.svg");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: #e5e5e5;
+    padding-bottom: 350px;
+    display: grid;
+    grid-template-columns: 1fr;
+    row-gap: 466px;
+  }
+
+  .headline-section {
+    width: 500px;
+    margin: 100px 0 100px 225px;
+  }
+
+  .hero {
+    width: 350px;
+    margin: 0;
+  }
+
+  .hero-details {
+    width: 750px;
+    height: inherit;
+    margin: 0;
+  }
+
+  .connect {
+    margin: 0;
+    margin: 20px 0 0;
+  }
+
+  .link {
+    padding: 0;
+    margin: 15px 0;
+  }
+
+  .projects {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    margin-top: 0;
+  }
+
+  .portfolio-projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    height: 500px;
+  }
+
+  .card-image {
+    display: none;
+  }
+
+  .card1 {
+    background-image: url("images/desktop-Snapshoot-Portfolio1.svg");
+    background-repeat: no-repeat;
+    background-position: 24px 24px;
+  }
+
+  .desktop-card-details {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 100%;
+    height: 70%;
+    padding-bottom: 80px;
```

Comment:
- [ ] [REQUIRED] Also, [this issue](https://github.com/Hamdullah-Andar/HamiPortfolio/pull/4#discussion_r1089165385) raised by the previous reviewer has not been fixed as I can see that some elements are overlapping each other.

Your design **(I have viewed the project with live server)**
![image](https://user-images.githubusercontent.com/84156714/215195771-53c1e460-1e6d-42b6-812c-7e47da46903c.png)